modules/installation/pages/hardware-requirements.adoc

```diff
@@ -154,15 +160,95 @@ For more information about inter-server synchronization (ISS), see xref:administ
 |===
 
 
-{productname} Server stores packages in the [path]``/var/spacewalk/`` directory.
-Repository synchronization fails if this directory runs out of disk space.
-You can estimate how much space the [path]``/var/spacewalk/`` directory requires based on the clients and repositories you plan to mirror.
-
 {productname} Proxy caches packages in the [path]``/var/cache/`` directory.
 If there is not enough space available in [path]``/var/cache/``, the proxy will remove old, unused packages and replace them with newer packages.
 
 As a result of this behavior:
 
-* The larger [path]``/var/cache/`` directory is on the {productname} Proxy, the less traffic there will be between it and the {productname} Server.
-* By making the [path]``/var/cache/`` directory on the {productname} Proxy the same size as [path]``/var/spacewalk/`` in the {productname} Server, you avoid a large amount of traffic after the first synchronization.
-* The [path]``/var/cache/`` directory can be small on the {productname} Server compared to {productname} Proxy. See estimation hints in the Server Hardware Requirements section.
+* The larger [path]``/var/cache/`` directory is on the proxy, the less traffic there will be between it and the {productname} Server.
+* By making the [path]``/var/cache/`` directory on the proxy the same size as [path]``/var/spacewalk/`` on the {productname} Server, you avoid a large amount of traffic after the first synchronization.
+* The [path]``/var/cache/`` directory can be small on the {productname} Server compared to the proxy.
+For a guide to size estimation, see the Server Hardware Requirements section.
+
+
+
+== Storage Devices and Permissions
+
+We recommend that the repositories and the database for {productname} are stored on separate storage devices.
+This will help to avoid data loss.
+You must set up the storage devices before you run the YaST {productname} setup procedure.
+
+{productname} requires three different volumes:
+
+* Database volume: [path]``/var/lig/pgsql``
+* Channel volume: [path]``/var/lib/spacewalk``
+* Cache: [path]``/var/cache``
+
+We recommend you use XFS as the filesystem type for all volumes.
+Additionally, consider using logical volume management (LVM) to manage the disks.
```

Comment:
I am considering a SUSE Manager running in the public cloud here.
XFS is sure an option _if_ the volumes above are non-root volumes (e.g. AWS EBS).
LVM: I do not think that is feasible or recommended in a public cloud setup.

@jgleissner do you have any input in this regard?

Reply:
@mbologna XFS is recommended in the public cloud too. We even use it for root volumes. LVM is of limited use in the cloud and not recommended.

Reply:
OK, so the hardware reqs page should be advice common to all installations, and we can keep cloud-specific advice to the pubcloud reqs section. With that in mind, it seem that I can keep the hardware reqs page as-is, but update the pubcloud reqs page to read "do not use LVM"?

Reply:
> OK, so the hardware reqs page should be advice common to all installations, and we can keep cloud-specific advice to the pubcloud reqs section. With that in mind, it seem that I can keep the hardware reqs page as-is, but update the pubcloud reqs page to read "do not use LVM"?

If the hardware requirements (this page) is common to all installations, then my suggestion is to be as general as possible: remove the LVM sentence from this and just add it to the "on-prem" documentation. So we do not have conflicting information between general and specific case setup.

Reply:
Updated.